### ISSUE TITLE
Xcode5 support build settings

### ISSUE THREAD

**Issue by JugglerShu** [XVimProject/XVim#448]
We need build configuration support both Xcode4 and 5.
We may need to create new target in the project.

Hopefully this includes following tasks.
- Create header files for Xcode5(IDEKit.h, DVTKit.h)
- Configuration easily switchable between Xcode5 and Xcode4
- Switch UUID in info.plist based on the configuration
- CI automation (travis or Xcode server?)
##

**Comment by daniel-beard** [comment]
@JugglerShu what's involved in creating the header files (IDEKit.h, DVTKit.h) ? 
Using class-dump directly gives me lots of build errors, is there some kind of manual processing needed?

**Comment by JugglerShu** [comment]
Yes, you need to manually add @class directives and reorder the declarations. One of the hardest and most boring part in this project hah.
